filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobPathBuilder.scala

```diff
@@ -12,12 +12,13 @@ import scala.language.postfixOps
 import scala.util.{Failure, Success, Try}
 
 object BlobPathBuilder {
-
+  val HOSTNAME_SUFFIX = ".blob.core.windows.net"
```

Comment:
I don't have a problem with it personally, but all-caps constants aren't a convention we really have in cromwell